notebooks/water_MD.ipynb

```diff
@@ -335,7 +335,7 @@
    "source": [
```

Comment:
<p>Thanks for extending this notebook! The only change I'd suggest is to not store the range of structures in a list as this can consume a lot of memory due to storing such objects in a list and because calling <code>get_structure()</code>  multiple times can slow down the notebook (because of multiple hdf5 access). So a better suggestion to go over the snapshots is: </p><p> </p><p><code>for pos in traj[::10]:</code></p><p><code>    struct.positions = pos</code></p><p><code>    neighbors = struct.get_neighbors(cutoff_radius=8)</code></p><p><code>    ..... # and so on</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron/pull/943/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Thanks a lot! Indeed, I was searching for a function to  avoid the looped get_structure() calls.